pkg/stanza/fileconsumer/config.go

```diff
@@ -88,7 +90,7 @@ type HeaderConfig struct {
 }
 
 // Build will build a file input operator from the supplied configuration
-func (c Config) Build(logger *zap.SugaredLogger, emit emit.Callback) (*Manager, error) {
+func (c Config) Build(set component.TelemetrySettings, logger *zap.SugaredLogger, emit emit.Callback) (*Manager, error) {
```

Comment:
Can we pass the logger first, so we're consistent with other signatures?

Reply:
Yeap. Will be fixed at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31618